xapian-applications/omega/query.cc

```diff
@@ -1566,6 +1566,24 @@ eval(const string &fmt, const vector<string> &param)
 		if (string_to_int(args[0]) > string_to_int(args[1]))
 		    value = "true";
 		break;
+	    case CMD_hash: {
+		string hash = args[0];
+		string data = args[1];
+		std::transform(hash.begin(), hash.end(), hash.begin(), ::tolower);
+		value = "";
+		if (hash == "md5") {
+		    trim_string(data);
```

Comment:
Yeah, this `trim_string()` function isn't useful anymore here so I'll go ahead and remove it.